install_config/registry/extended_registry_configuration.adoc

```diff
@@ -303,9 +303,30 @@ are supported:
 *  link:https://docs.docker.com/registry/storage-drivers/s3/[S3]. Learn more about https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/s3.md#cloudfront-as-middleware-with-s3-backend[CloudFront configuration].
 * link:https://docs.docker.com/registry/storage-drivers/swift/[OpenStack Swift]
 * link:https://docs.docker.com/registry/storage-drivers/gcs/[Google Cloud Storage (GCS)]
+* link:https://docs.docker.com/registry/storage-drivers/azure/[Microsoft Azure]
+* link:https://docs.docker.com/registry/storage-drivers/oss/[Aliyun OSS]
 
 link:https://docs.docker.com/registry/configuration/#maintenance[General registry storage configuration options] are supported.
 
+[NOTE]
+====
+The list includes just the filesystems that need to be configured in the
+registry's config file with an exception of filesystem which is the default.
+There are other storage options like:
+
+* xref:../../install_config/storage_examples/gluster_backed_registry.adoc#dedicated-glusterfs-dynamic-example-overview[GlusterFS
+Storage]
+* xref:../../install_config/persistent_storage/persistent_storage_ceph_rbd.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[Ceph
+Rados Block Device]
+* xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-persistent-storage-ceph-rbd[And
+more]. See
+xref:../../install_config/storage_examples/index.adoc#install-config-storage-examples-index[examples]
+as well.
+
+Using these storage options requires the
+link:https://docs.docker.com/registry/storage-drivers/filesystem[filesystem
```

Comment:
You must configure the link:https://docs.docker.com/registry/storage-drivers/filesystem[filesystem driver] to use these storage options.